api/tests/identity-access-management/unit/domain/services/reset-password.service.test.js

```diff
@@ -13,10 +13,10 @@ describe('Unit | Identity Access Management | Domain | Service | reset-password'
     beforeEach(function () {
       sinon.stub(jsonwebtoken, 'sign');
       randomGeneratedString = 'aaaaaa';
-      sinon.stub(crypto, 'randomBytes').returns(randomGeneratedString);
+      sinon.stub(crypto, 'randomBytes').yields(null, 'aaaaaa');
```

Comment:
Pour renforcer la consistance du test (ou alors virer la variable `randomGeneratedString` si elle ne sert plus à rien selon toi)

```suggestion
      sinon.stub(crypto, 'randomBytes').yields(null, randomGeneratedString);
```